util/util.go

```diff
@@ -90,3 +93,18 @@ func RecoverIndexFromNodeName(name string) int {
 	}
 	return out
 }
+
+func GenerateFieldSelectorFromWorkflowName(wfName string) string {
```

Comment:
Can we add unit tests for these two functions?

Reply:
Absolutely. Which file is appropriate for these tests?

Reply:
You can create a new `util/util_test.go` file if one already doesn't exist